docs/src/TerminalVelocityComparisons.jl

```diff
@@ -0,0 +1,229 @@
+import Plots as PL
+
+import Thermodynamics as TD
+import CloudMicrophysics as CM
+import CLIMAParameters as CP
+#using SpecialFunctions
+
+const CMT = CM.CommonTypes
+const CM2 = CM.Microphysics2M
+const CMP = CM.Parameters
+const CH = CM.TerminalVelocityChen2022
+
+const APS = CMP.AbstractCloudMicrophysicsParameters
+include(joinpath(pkgdir(CM), "test", "create_parameters.jl"))
+FT = Float64
+toml_dict = CP.create_toml_dict(FT; dict_type = "alias")
+const param_set = cloud_microphysics_parameters(toml_dict)
+thermo_params = CMP.thermodynamics_params(param_set)
+
+const liquid = CMT.LiquidType()
+const ice = CMT.IceType()
+const rain = CMT.RainType()
+const snow = CMT.SnowType()
+const SB2006 = CMT.SB2006Type()
+const Chen2022 = CMT.Chen2022Type()
+
```

Comment:
We use the velocities of individual drops only for visualisation/documentation. Therefore, could we define the Chen2022 individual velocity here instead of in the source code? Also, could we also add the 1-moment terminal velocity for comparison (I think it will be quite different than the other two)